src/aleph/chains/ethereum.py

```diff
@@ -304,18 +305,24 @@ async def ethereum_packer(config):
 
         if len(messages):
             content = await get_chaindata(messages, bulk_threshold=200)
-            response = await run_in_executor(
-                None,
-                broadcast_content,
-                config,
-                contract,
-                web3,
-                account,
-                int(gas_price * 1.1),
-                nonce,
-                content,
-            )
-            LOGGER.info("Broadcasted %r on %s" % (response, CHAIN_NAME))
+            try:
+                response = await run_in_executor(
+                    None,
+                    broadcast_content,
+                    config,
+                    contract,
+                    web3,
+                    account,
+                    int(gas_price * 1.1),
+                    nonce,
+                    content,
+                )
+                LOGGER.info("Broadcasted %r on %s" % (response, CHAIN_NAME))
+            except Exception:
```

Comment:
Can you avoid catching _any_ exception silently ? 
Whitelist the exception classes and/or log them so we can catch them in Sentry.

Reply:
After some tests, an insufficient balance will result in a ValueError. I think the safest approach is to catch Exception, drop the file and then re-raise the exception.

Reply:
OK with the re-raise